Calculator.rb

```diff
@@ -0,0 +1,83 @@
+# command line calculator
+# array of operators for later use
+op_arr = ["add", "+", "addition", "subtract", "subtraction", "-", "multiply", "*", "multiplication", "divide", "division", "/", "exponent", "^", "modulo", "remainder", "%"]
+
+# array to store user input numbers
+number = []
+
+# methods for addition, subtraction, multiplication, division, exponents, and modulus
+
+def add(a, b)
+  return a + b
+end
```

Comment:
I like that you've split these out into separate methods! It may seem a little pedantic for things like + and -, but for more complicated methods (for example checking for zero before doing division) it results in much cleaner code.